### PR TITLE
Feature/218

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
[Fecfile #218](https://fecgov.atlassian.net/browse/FECFILE-218?atlOrigin=eyJpIjoiZmUyMGFhNjY3N2RkNDk4NjhiMTE4YmQzMTU1YmJkMzkiLCJwIjoiaiJ9)

### CODE REVIEW

File: django-backend/fecfiler/openfec/views.py [hunk 1]
```diff
@@ -50,15 +50,17 @@ def retrieve_recent_f1(committee_id):
...
     headers = {"Content-Type": "application/json"}
     params = {
         "api_key": settings.FEC_API_KEY,
         "committee_id": committee_id,
-        "sort": "-receipt_date",
-        "form_type": "F1",
     }
-    endpoints = [f"{settings.FEC_API}efile/filings/", f"{settings.FEC_API}filings/"]
+    endpoints = [
+        f"{settings.FEC_API}efile/form1/",
```
Comment (toddlees) [inline django-backend/fecfiler/openfec/views.py:61]: can you change this to ```f"{settings.FEC_API}efile/form1/?sort=-load_timestamp```
there can be multiple versions of the same f1, and we need the most recent.

**Review by toddlees — CHANGES_REQUESTED** [review]

**Comment by toddlees** [comment]
We're gonna need some follow up tickets.  This method `retrieve_recent_f1` is used in two places.
1) registering a committee account 
  - issue:  the f1 filing is used to get the email.  The second openfec endpoint `committee/{committee_id}` doesn't contain an email so this only works for that use case if the f1 filing is available in the first endpoint
2) creating the button that links to the f1 pdf on the committee info page
  - issue: here the f1 filing is used to get the pdf url.  again, the second endpoint doesn't have that info.  so it doesn't serve it's purpose if the filing is older than 6 months

File: django-backend/fecfiler/openfec/views.py (hunk 1 above)
Comment (Elaine-Krauss-TCG) [reply django-backend/fecfiler/openfec/views.py:61]: Sorry, I didn't see this.

If you look at the [https://api.open.fec.gov/developers/#/efiling/get_v1_efile_filings_](docs), the default sorting is `-receipt_date` which should satisfy that requirement.

**Comment by Elaine-Krauss-TCG** [comment]
> We're gonna need some follow up tickets. This method `retrieve_recent_f1` is used in two places.
> 
>     1. registering a committee account
> 
>     * issue:  the f1 filing is used to get the email.  The second openfec endpoint `committee/{committee_id}` doesn't contain an email so this only works for that use case if the f1 filing is available in the first endpoint

The `committee/{committee_id}` [endpoint](https://api.open.fec.gov/developers/#/committee/get_v1_committee__committee_id__) does contain the email field, but it does _not_ contain the F1 filing url.

**Pull request merged**